prov/gni/src/gnix_msg.c

```diff
@@ -1577,7 +1577,7 @@ ssize_t _gnix_send(struct gnix_fid_ep *ep, uint64_t loc_addr, size_t len,
 	}
 
 	if ((flags & FI_INJECT) && (len > GNIX_INJECT_SIZE)) {
-		GNIX_INFO(FI_LOG_EP_DATA,
+		GNIX_DEBUG(FI_LOG_EP_DATA,
```

Comment:
I think this should remain GNIX_INFO, because it's the type of message that could help an end user.